examples/pytorch_lightning_distributed/msn.py

```diff
@@ -92,11 +93,11 @@ def configure_optimizers(self):
 pascal_voc = torchvision.datasets.VOCDetection(
     "datasets/pascal_voc", download=True, target_transform=lambda t: 0
 )
-dataset = LightlyDataset.from_torch_dataset(pascal_voc)
+dataset = LightlyDataset.from_torch_dataset(pascal_voc, transform=MSNTransform())
```

Comment:
Here, too: Do we have to set `MSNTransform(384)` or not?

Reply:
No, see above.